osu.Game/Screens/Edit/Timing/MetronomeDisplay.cs

```diff
@@ -232,6 +236,24 @@ private void load(AudioManager audio)
 
         private ScheduledDelegate? latchDelegate;
 
+        private bool spedUp;
+
+        private bool updateDivisor()
+        {
+            int divisor = 1;
+
+            if (spedUp)
+                divisor = beatDivisor.Value % 3 == 0 ? 3 : 2;
```

Comment:
dunno why the only thing this checks is divisibility by 3, if someone has 1/7 beat snap does it make sense to double-time it?

i'd say double-time if snap divisible by 2, triple-time if divisible by 3, and nothing otherwise